bootstrapvz/plugins/puppet/README.rst

```diff
@@ -1,24 +1,111 @@
 Puppet
 ------
 
-Installs `puppet <http://puppetlabs.com/>`__ and optionally applies a
+Installs `puppet version 4 <http://puppetlabs.com/>` PC1 From the site 
+repository `<http://apt.puppetlabs.com/>` and optionally applies a
 manifest inside the chroot. You can also have it copy your puppet
 configuration into the image so it is readily available once the image
 is booted.
 
-Keep in mind that when applying a manifest, the system is in a chrooted
-environment. This can prevent daemons from running properly (e.g.
-listening to ports), they will also need to be shut down gracefully
-(which bootstrap-vz cannot do) before unmounting the volume. It is
-advisable to avoid starting any daemons inside the chroot at all.
+Rationale and use case
+~~~~~~~~~~~~~~~~~~~~~~
+
+You want to use this plugin when you wish to create an image and to be able to
+manage that image with Puppet. You have a Puppet 4 setup in mind and thus you 
+want the image to contain the puppet agent software from the puppetlabs repo. 
+You want it to almost contain everything you need to get it up and running 
+This plugin does just that!
+While you're at it, throw in some modules from the forge as well!
+Want to include your own modules? Include them as assets!
+
+This plugin attempts to create more abstraction to the bootstrap-vz build 
+process by creating a robust plugin that enables you to use the most up to date
+Puppetlabs software with your images fresh from the build-oven.
+Before, when using this plugin, you were using the puppet agent software 
```

Comment:
My apologies, i've added a short desciption in the changelog and removed these lines from the readme